prime-router/metadata/hl7_mapping/ORU_R01/base/order-observation.yml

```diff
@@ -2,10 +2,6 @@ name: ORU-R01-order-observation
 constants:
   hl7Order: /PATIENT_RESULT/ORDER_OBSERVATION(%{orderIndex})
 elements:
-  - name: request-set-ID
```

Comment:
This was also duplicated in the observation-request schema. It seems more appropriate to stay there.